modules/fbx/config.py

```diff
@@ -1,4 +1,5 @@
 def can_build(env, platform):
+    env.module_add_dependencies("fbx", ["gltf"])
```

Comment:
This makes the tight coupling between fbx and gltf explicit. `fbx` can't be compiled without `gltf` (but `gltf` can be compiled without `fbx`, after I moved FBX2glTF here due to the introduced coupling).